### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by SolarAquarion** [budgie-desktop/budgie-desktop#958]
**Operating System**: Arch Linux
**Budgie version**:  10.3.1

[20/232] Compiling Vala source ../src/lib/ani.../src/lib/shadow.vala ../src/lib/manager.vala.
FAILED: src/lib/toplevel@sta/animation.c src/lib/toplevel@sta/toplevel.c src/lib/toplevel@sta/shadow.c src/lib/toplevel@sta/manager.c src/lib/toplevel.h src/lib/toplevel.vapi 
valac -C --debug -lm --pkg gtk+-3.0 --pkg libpeas-1.0 --pkg libpeas-1.0 --pkg gtk+-3.0 -d src/lib/toplevel@sta --library=toplevel -H src/lib/toplevel.h --use-header --vapi ../toplevel.vapi --pkg budgie-1.0 --pkg gio-unix-2.0 --pkg gtk+-3.0 --pkg libpeas-1.0 --vapidir /home/solaraquarion/build/budgie-desktop-git/src/budgie-desktop-git/src/plugin -H BudgieToplevel.h ../src/lib/animation.vala ../src/lib/toplevel.vala ../src/lib/shadow.vala ../src/lib/manager.vala
Unknown option -lm
Run 'valac --help' to see a full list of available command line options.
[29/232] Compiling Vala source ../src/appsys/AppSystem.vala.
../src/appsys/AppSystem.vala:240.31-240.43: warning: Gdk.X11Window has been deprecated since vala-0.24. Use Gdk.X11.Window
Compilation succeeded - 1 warning(s)
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Aborting...

rc/appsys/appsys@sta/AppSystem.c'
src/appsys/appsys@sta/AppSystem.c:17:10: fatal error: BudgieAppSystem.h: No such file or directory
 #include "BudgieAppSystem.h"
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.

**Comment by fossfreedom** [comment]
yep - same as this https://github.com/budgie-desktop/budgie-desktop/issues/951

TL;DR; your distro has got a newer meson version that is broken - needs to be flagged upstream https://github.com/mesonbuild/meson

**Comment by ikeydoherty** [comment]
If its possible to get your distro to patch or roll back, do, and make sure to add weight to necessary upstream meson issues

**Comment by Mailaender** [comment]
Same problem on openSUSE with meson 0.41.1 at the moment. According to https://github.com/mesonbuild/meson/pull/1970 this will be fixed with meson 0.41.2 again.

**Comment by ikeydoherty** [comment]
Hope so.
